docs/api-spec.md

```diff
@@ -244,7 +244,8 @@ List responses have the following fields (based on [`meta.v1/ListMeta`](https://
 |---------------|------------|-------------|
 | `name`        | string     | REQUIRED    |
 | `description` | string     | REQUIRED    |
-| `type`        | Enum: <br>- `"string"` (default) <br>- `"array"` | REQUIRED |
+| `type`      | Enum:<br>- `"string"` (default)<br>- `"array"` <br>- `"object"` | REQUIRED for `string` and `array`type <br> RECOMMENDED for `object` type |
```

Comment:
I think it would make sense to phrase this as the "type" field is required, and the values "string" and "array" for this field are required, and the value "object" is recommended.